src/xeus_sqlite_interpreter.cpp

```diff
@@ -38,15 +46,19 @@ namespace xeus_sqlite
         std::string path_to_db = "/Users/mariana/Development/jupyter-ecosystem/xeus-sqlite/chinook.db";
         SQLite::Database db(path_to_db, SQLite::OPEN_READWRITE|SQLite::OPEN_CREATE);
 
+        //if user wants to open a db or some other specific case
+        // parse_code(code);
+
+        //else execute normal sqlite code
         SQLite::Statement query(db, code);
         std::stringstream query_result("");
 
         //Iterates through the columns and prints the columns
         while (query.executeStep())
         {
             for(int column = 0; column < query.getColumnCount(); column++) {
-                std::string name = query.getColumn(column);
-                query_result << name << std::endl;
+                std::string col_content = query.getColumn(column);
+                query_result << col_content << std::endl;
```

Comment:
endline after every _column_?

maybe you want to check out `tabulate` for C++ (https://github.com/p-ranav/tabulate) 
if you want I can help you make a conda-forge recipe :)

Reply:
This was just to have some legible printed in the screen, I'm not worried yet with how it looks. But tabulate looks pretty cool! Thanks :)

Reply:
but I think for the short term, maybe just put a ` | ` instead of endline, and then an endline after the row so you get something that resembles a table a little bit :)